diff --git a/node_modules/lzma/.bun-tag-427a975232d7fd5a b/.bun-tag-427a975232d7fd5a
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/index.js b/index.js
index 3ddd619efe1dc757dd4141fc086783ffec44a484..8329e0155c028d7eac1de76768f4667084ef2d98 100644
--- a/index.js
+++ b/index.js
@@ -4,7 +4,7 @@ var lzma;
 
 function load_lzma()
 {
-    return require(require("path").join(__dirname, "src" ,"lzma_worker.js")).LZMA_WORKER;
+    return require("./src/lzma_worker.js").LZMA_WORKER;
 }
 
 lzma = load_lzma();
diff --git a/package.json b/package.json
index a2091e4067098045d054a333d01eaf07bd3f7ca0..1feaa199ebcceae0a7140a32612bf66520cb3c31 100644
--- a/package.json
+++ b/package.json
@@ -4,6 +4,7 @@
   "description": "A JavaScript implementation of the Lempel-Ziv-Markov (LZMA) chain compression algorithm",
   "version": "2.3.2",
   "homepage": "http://nmrugg.github.com/LZMA-JS/",
+  "main": "index.js",
   "repository": {
     "type": "git",
     "url": "git://github.com/nmrugg/LZMA-JS.git"
